cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunTestsTask.java

```diff
@@ -531,6 +555,7 @@ private int runTestSuit(Path testCachePath, Target target, Package currentPackag
         // Adds arguments to be read at the Test Runner
         // Index [0 - 3...]
         cmdArgs.add(target.path().toString());
+        cmdArgs.add(jacocoAgentJarPath);
```

Comment:
Here we are adding the `jacocoAgentJarPath` regardless of wether the code coverage or report flag is provided. 

Would it make sense to have the jacocoAgentJarPath be set only when the coverage is requested like how it used to be? Or would that change break the code coverage?

Reply:
The condition used to be `if (coverage is enabled) then set jacoco jar path`.  But now we set the jar path to jacoco everytime. Would declaring the string variable outside the condition and initializign it inside work?

Reply:
Fixed the code as if codecoverage is false it will be empty string.